markdown.js

```diff
@@ -0,0 +1,103 @@
+const visit = require('unist-util-visit');
+var remark = require('remark');
+var parse = require('remark-parse');
+var html = require('remark-html');
+var fs = require('fs');
+
+var file = fs.readFileSync('./src/content/plugins/index.md', { encoding: 'utf8' });
+
+remark()
+  .use(parse)
+  .use(md)
+  .use(html)
+  .process(file, function(err, file) {
+    if (file) {
+      console.log(file.contents);
+    }
+  });
+
+module.exports = md;
+
+function addClassName(node, className) {
+  node.data = {};
```

Comment:
since you are creating the whole data object on the node why not do the scaffolding in one declaration:
```
node.data = {
  hProperties: { className }
};
```

Reply:
Will be removed